backends/ebpf/build_libbpf

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+#!/usr/bin/env python2
```

Comment:
Is it intentional that both /usr/bin/env lines are in this script?

Reply:
And should this also perhaps explicitly call out Python 3, to avoid introducing new Python 2 scripts in 2020?